testkit/build.sbt

```diff
@@ -1,7 +1,16 @@
-import Dependencies._
+import Dependencies.Version._
 
 name := "pureconfig-testkit"
 
-libraryDependencies ++= Seq(scalaTest, scalaCheck, scalaTestPlusScalaCheck)
+crossScalaVersions := Seq(scala211, scala212, scala213, scala30)
+
+libraryDependencies ++= Seq(
+  Dependencies.scalaTest,
+  Dependencies.scalaCheck.withDottyCompat(scalaVersion.value),
+  Dependencies.scalaTestPlusScalaCheck.intransitive().withDottyCompat(scalaVersion.value)
```

Comment:
Why are we doing this? Can you add a comment explaining why this is needed?

Reply:
We need to avoid having scalactic and scalatest-core pulled both for Scala 3.x (via the `scalaTest` dependency) and for Scala 2.x (via the `scalaTestPlusScalaCheck` dependency).